toolchain/src/integration-test/java/org/archcnl/toolchain/ToolchainIT.java

```diff
@@ -0,0 +1,92 @@
+package org.archcnl.toolchain;
+
+import static org.junit.Assert.assertTrue;
+
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Arrays;
+
+import org.apache.jena.ontology.OntModel;
+import org.apache.jena.ontology.OntModelSpec;
+import org.apache.jena.query.Query;
+import org.apache.jena.query.QueryExecution;
+import org.apache.jena.query.QueryExecutionFactory;
+import org.apache.jena.query.QueryFactory;
+import org.apache.jena.rdf.model.ModelFactory;
+import org.archcnl.stardogwrapper.api.StardogDatabaseAPI;
+import org.archcnl.stardogwrapper.impl.StardogDatabase;
+import org.junit.Test;
+
+public class ToolchainIT {
```

Comment:
It would be nice to spell out the abbreviation "IT" for increased clarity. A longer name does not matter since we never need to refer to this class anywhere else.

Reply:
In the end, this is some convention issue which should be discussed in the team. In my opinion, extending `IT` to `IntegrationTest` would be redundant because integration tests are already located in `src/integration-tests`.

Reply:
Ok, so this can be left as is.